linalg/solvers.cpp

```diff
@@ -2615,13 +2615,13 @@ struct WeightMinHeap
       c.pop_back();
       // Mark as removed
       loc[i] = -1;
-      if (c.empty()) { return i; }
```

Comment:
Could you please fix other conversions between `size_t` and `int` around here (methods of `WeightMinHeap`)? `pos` is `size_t`, while `loc` stores ints. It shows as `_Ty` in the log, but actually it is that `int`.

Reply:
Done in https://github.com/mfem/mfem/pull/4562/commits/e90cff6413a72e9c96cd12547e9e18e3d88b2746

Reply:
Great, I see even some bonuses here 👍